frontend/src/lib/utils.tsx

```diff
@@ -737,7 +738,25 @@ export const formatDateRange = (dateFrom: dayjs.Dayjs, dateTo: dayjs.Dayjs, form
     return `${dateFrom.format(formatFrom)} - ${dateTo.format(formatTo)}`
 }
 
-export const dateMapping: DateMappingOption[] = [
+// Hourly date mappings for real time analytics. Only shown on Trends at the moment.
+// Could potentially be expensive to compute.
+export const realTimeDateMapping: DateMappingOption[] = [
+    {
+        key: 'Last Hour',
+        values: ['-1h'],
+        getFormattedDate: (date: dayjs.Dayjs): string =>
+            formatDateRange(date.subtract(1, 'h'), date, DATE_FORMAT_WITH_TIME),
+        defaultInterval: 'minute',
+    },
+    {
+        key: 'Last Three Hours',
```

Comment:
This should follow the style we already use:
```suggestion
        key: 'Last 3 hours',
```